pkg/github/tokensource.go

```diff
@@ -17,12 +17,16 @@ package github
 import (
 	"context"
 	"fmt"
+	"os"
 	"strconv"
 
 	"github.com/abcxyz/pkg/githubauth"
 	"github.com/abcxyz/team-link/pkg/credentials"
 )
 
+// DefaultStaticTokenEnvVar is where we read default github token from.
+const DefaultStaticTokenEnvVar = "TEAM_LINK_GITHUB_TOKEN" // #nosec G101
```

Comment:
Could you add a code comment explaining why we need the `#nosec`?

Reply:
added